runtime/browser/xwalk_utils.cc

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/browser/xwalk_utils.h"
+
+namespace xwalk_utils {
+
+bool IsLocalHost(const GURL& url) {
+  std::string host = url.host();
+  return host.compare("localhost") ||
```

Comment:
Nope, they are not related. @rakuco moved the bot to Portland and we experiencing flakiness on the test (though it was flaky before).